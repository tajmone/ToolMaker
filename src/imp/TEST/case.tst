%%SET foo(fie)
%%SET FOO(FUM)
%%IF(foo(fie) AND NOT foo(FIE) AND FOO(FUM) AND NOT FOO(fum))
OK!
%%ELSE
Error!
%%ENDIF
