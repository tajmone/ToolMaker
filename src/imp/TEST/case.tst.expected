########## case ##########
%%SET foo(fie)
%%SET FOO(FUM)
%%IF(foo(fie) AND NOT foo(FIE) AND FOO(FUM) AND NOT FOO(fum))
OK!
%%ELSE
Error!
%%ENDIF
########## ../imp case.tst >case.tst.tmp; cat case.tst.tmp; rm case.tst.tmp ##########
OK!
